package.json

```diff
@@ -84,16 +83,17 @@
     "@types/pluralize": "^0.0.30",
     "@types/prompts": "^2.4.2",
     "@types/semver": "^7.3.12",
-    "@types/tmp": "^0.2.3",
```

Comment:
Did it surface these as unused dependencies?

Reply:
correct!